src/ItemList.js

```diff
@@ -128,58 +160,87 @@ function ItemList(props) {
       {props.error && !props.loading && <strong>Error: {props.error}</strong>}
       {props.list && props.list.length === 0 && (
         <>
-          <p>Your shopping list is currently empty.</p>
-          <button onClick={redirect}>Add Item</button>
+          <StyledNoItems>
+            <Typography variant="h1">Your list</Typography>
+            <StyledP>
+              <Typography variant="p">
```

Comment:
```suggestion
              <Typography paragraph>
```
https://material-ui.com/api/typography/#main-content

Reply:
Then you can probably get rid of the `StyledP`